sdk/ml/azure-ai-ml/azure/ai/ml/entities/_job/parameterized_command.py

```diff
@@ -68,7 +68,7 @@ class ParameterizedCommand:
 
     def __init__(
         self,
-        command: Optional[str] = "",
+        command: Union[str, None] = "",
```

Comment:
Same as my other comment. `Optional[str]` is the same as `Union[str, None]`. 

Based on my understanding of this class, it's used a representation of commands used as part of a Job type object as an attribute. Those Job type classes default to None for the ParameterizedCommand attribute often (e.g. https://github.com/Azure/azure-sdk-for-python/blob/930d75e98460ee755a0f3710b83fc84f6b8701d7/sdk/ml/azure-ai-ml/azure/ai/ml/entities/_job/sweep/sweep_job.py#L146C1-L147C1)  which is why if this class isn't `Optional`, mypy complains.

@annatisch What do you suggest here?

Reply:
Changed it back to Optional[str]